pkg/server/backuprestoreserver.go

```diff
@@ -196,6 +201,9 @@ func (b *BackupRestoreServer) runServer(ctx context.Context, restoreOpts *brtype
 					b.logger.Info("backup-restore stops leading...")
 				}
 				handler.SetSnapshotterToNil()
+
+				// To do: For Multi-node etcd HTTP status need to be set to `StatusServiceUnavailable` only when backup-restore is in "StateUnknown".
```

Comment:
```suggestion
				// TODO: @ishan16696  For Multi-node etcd HTTP status need to be set to `StatusServiceUnavailable` only when backup-restore is in "StateUnknown".
```
This makes `TODO`s easily trackable. Can you also please raise an issue for this if necessary (so that it's not forgotten later)? Thanks.

Reply:
created this issue https://github.com/gardener/etcd-druid/issues/288 describing the readinessProbe for multi-node etcd.